debezium-connector-postgres/src/test/resources/init_pgvector.ddl

```diff
@@ -0,0 +1,7 @@
+-- noinspection SqlNoDataSourceInspectionForFile
+
+-- Separate file because pgvector is tested since PostgreSQL 16
```

Comment:
Test annotation says PostgreSQL 15+ while this says PostgreSQL 16+; which is the baseline?